pems-server/src/main/java/org/maroubra/pemsserver/api/models/sensors/responses/SensorHistoryResponse.java

```diff
@@ -1,4 +1,37 @@
 package org.maroubra.pemsserver.api.models.sensors.responses;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableMap;
+import org.maroubra.pemsserver.monitoring.SensorLog;
+
+import java.util.ArrayList;
+import java.util.List;
+
 public class SensorHistoryResponse {
```

Comment:
I'm not sure the resulting JSON is what we would want? Can you give an example JSON you'd expect from this? Are we matching type, value and time by index?

Reply:
currently, the JSON  will look like this, the reason I did that was to accommodate for values from the accelerometer.
{
   "valueName":[
      [
         "BAT"
      ],
      [
         "accel-x",
         "accel-y",
         "accel-z"
      ]
   ],
   "value":[
      [
         91
      ],
      [
         0.9,
         5,
         0.3
      ]
   ],
   "timeStamp":[
      "2017-10-28T10:06:11.859",
      "2017-10-29T10:06:11.859"
   ]
}
However we can make it look like this by changing add to addAll.
{
   "valueName":[
      "BAT",
      "accel-x",
      "accel-y",
      "accel-z"
   ],
   "value":[
      91,
      0.9,
      5,
      0.3
   ],
   "timeStamp":[
      "2017-10-28T10:09:08.318",
      "2017-10-29T10:09:08.321"
   ]
}

Reply:
or I can put in checks and combine both of them together.